### ISSUE TITLE
Find a fix to tf.nn.ctc_beam_search_decoder()'s treatment of repeated characters

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#53]

**Comment by kdavis-mozilla** [comment]
It seems like there might be a problem with `tf.nn.ctc_beam_search_decoder()` and its treatment of repeated characters.

The sequence `A` `L` `L` `*` `L` (where '`*`' is the blank label) will become 
- `A` `L` if `merge_repeated=True`.
- `A` `L`  `L`  `L` if `merge_repeated=False`.

So, unless the `L`'s happen to align with the frames, it is hard to impossible to get words with characters repeated. A solution for this needs to be found. 

Currently I assume the language model fixes this, but this has to be verified when the language model is in place. Until then this seems like a bug.
